docs/opts.rst

```diff
@@ -92,6 +92,33 @@ Data handling options
 
   .. note:: The index *sets* are the same in the two ``modeord`` choices; their ordering differs only by a cyclic shift. The FFT ordering cyclically shifts the CMCL indices $\mbox{floor}(N/2)$ to the left (often called an "fftshift").
 
+.. _sionly:
+
+**spreadinterponly**: [only has effect for type 1 or 2.] For experts only!
+If ``0`` do
+the NUFFT as intended.  If ``1``, omit the FFT and deconvolution
+(diagonal division by kernel Fourier transform) steps, thus returning
+*garbage answers as a NUFFT*, but allowing experts to perform solely
+spreading (if type 1) or solely interpolation (if type 2) by hijacking
+the usual FINUFFT API.  The spreading is onto the grid of the
+user-given size (``N1`` in x, ``N2`` in y, etc), with grid points
+located at coordinates $\{-\pi, -\pi+h, \dots, \pi-h\}$ in each
+dimension, where $h = 2\pi/N$ is the spacing for that dimension ($N$
+here meaning ``N1``, etc). Interpolation is from that same grid.  The
+kernel (width and shape parameter) is determined by ``tol`` and
+``opts.upsampfac``, just as it would be in an actual NUFFT. Note that
+the upsampling factor here only controls the kernel; the grid size
+never differs from ``N1``, etc.  The kernel is not directly
+accessible, leaving the user to figure out how to make use of this
+interface to extract the actual kernel function.  This provides a
+convenient (if hacky) interface to our ``spreadinterp`` module
+(including looping over multiple vectors, if ``ntransf>1``).  The
+known use-case here is estimating so-called density compensation,
+conventionally used in MRI (see `MRI-NUFFT
+<https://mind-inria.github.io/mri-nufft/nufft.html>`_), although it
+might also be useful in spectral Ewald.
```

Comment:
Looking at the rendered docs here: https://github.com/flatironinstitute/finufft/blob/spreadinterponly/docs/opts.rst

modeord has a nice itemize. I think is worth doing the same.

Reply:
Maybe: *if ``1``: It does not perform a NUFFT!* 

hijacking -> adapting

Reply:
*if hacky* -> unconventional | unorthodox